WordPress/Classes/WordPress.xcdatamodeld/WordPress 45.xcdatamodel/contents

```diff
@@ -113,6 +113,7 @@
         <attribute name="options" optional="YES" attributeType="Transformable">
             <userInfo/>
         </attribute>
+        <attribute name="planID" optional="YES" attributeType="Integer 64" defaultValueString="0" syncable="YES"/>
```

Comment:
Just in case, I'd remove the default value. If there's no plan, we want planID to be `nil`, not `0`